core/task_medium_test.go

```diff
@@ -72,8 +145,16 @@ func okRoutine(sch schedule.Schedule,
 	return nil, nil
 }
 
-func TestUnmarshalBodyTask(t *testing.T) {
+func createTaskFile(name string, content []byte) {
+	f, _ := os.Create(name)
```

Comment:
The function should return the error, so you can test if error is `nil` with the convey framework.